Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -14,43 +12,27 @@
 internal partial class CoreTests
 {
     const string TestCategory = "ProjectWideActions";
-    const string TestAssetPath = "Assets/TestInputManager.asset";
-    string m_TemplateAssetPath;
 
-#if UNITY_EDITOR
     const int initialActionCount = 2;
     const int initialMapCount = 1;
-#else
-    const int initialActionCount = 19;
-    const int initialMapCount = 2;
-#endif
 
     [SetUp]
     public override void Setup()
     {
-        // @TODO: Currently we can only inject the TestActionsAsset in PlayMode tests.
-        // It would be nice to be able to inject it as a Preloaded asset into the Player tests so
-        // we don't need different tests for the player.
-        // This also means these tests are dependant on the content of InputManager.asset not being changed.
-#if UNITY_EDITOR
-        // This asset takes the place of ProjectSettings/InputManager.asset for the sake of testing, as we don't
+        // This asset takes the place of the project wide actions asset for the sake of testing, as we don't
```

Comment:
Changes here do not relate to Typesafe API and could have been done separately against develop on its own PR?